src/views/catalog/templatescatalog/utils/helpers.ts

```diff
@@ -28,12 +28,14 @@ export const filterTemplates = (templates: V1Template[], filters: TemplateFilter
 
       const defaultVariantFilter = filters?.onlyDefault ? isDefaultVariantTemplate(tmp) : true;
 
+      const userFilter = filters?.onlyUser ? !isDefaultVariantTemplate(tmp) : true;
```

Comment:
Could this work also?
!filters.onlyUser || !isDefaultVariantTemplate

Reply:
Yes, this would work, too. However, I used ternary operator to make the statement more readable, understandable and consistent with other ones, for example look at `defaultVariantFilter` above. I believe, it was implemented and approved in this form for good reasons. Or not?

Reply:
well for me its not clearer , as it doing : true which can be simply avoided as it not really needed, also for the lines above and below

Reply:
Well, interesting that somebody approved such code in the past here.

Reply:
sometimes we approve and then improve later